### ISSUE TITLE
Some data not updating without a refresh

### ISSUE THREAD

**Issue by ville-j** [elmadev/elmaonline-site#108]
You may also call this a feature rather than a bug, but since the data queries are cached by default the data on the site is not always up-to-date (without a page refresh). This is a bit misleading especially considering current battle, latest battles and the battle listing of current date. It is worth considering on which occasions the caching could be disabled.

**Comment by sunehs** [comment]
Yeah I noticed that aswell once we pushed to live. Will need to look into how exactly the caching works. Another option is using the refetch function in relevant places on page change and/or periodically. Currently use that after uploading a replay to update the list: https://github.com/elmadev/elmaonline-site/blob/dev/src/routes/home/Home.js#L148
